pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -132,9 +132,8 @@ func Apply(target *deployments.Target) error {
 				return err
 			}
 			err = target.Apply(deployments.KubernetesBaseOSConfiguration{
-				KubeadmVersion:    nodeVersionInfoUpdate.Update.APIServerVersion.String(),
```

Comment:
We might want to revisit this struct later

Reply:
Since we are not merging this immediately, do you think you could rename `KubeadmVersion` and `KubernetesVersion` to something more meaningful now that we are not forcing explicit package installs? something like `CurrentVersion` and `NewVersion` or something like that?

Reply:
updated in a separate commit